library/ssl_cookie.c

```diff
@@ -44,7 +44,7 @@
  * available. Try SHA-256 first, 512 wastes resources since we need to stay
  * with max 32 bytes of cookie for DTLS 1.0
  */
-#if defined(MBEDTLS_SHA256_C)
+#if defined(MBEDTLS_SHA224_C)
```

Comment:
Since 224 and 256 are bound to each other, we can expect `MBEDTLS_SHA224_C` to be defined. At least until next library version. Are you sure this should be changed back to `#if defined(MBEDTLS_SHA256_C)` ?

Reply:
Ok, let's keep 224 for now.